### PR TITLE
Navi-45/feat/login-page

### PR DESCRIPTION
**Pull request by Dachikkinho**

### CODE REVIEW

File: app/Login/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<LoginForm>();
+
+  const [showPassword, setShowPassword] = useState(false);
+
+  const onLoginFinished = (values: LoginForm) => {
+    console.log("FORM SUBMITTED", values);
```
Comment (lursmana1) [inline app/Login/Login.tsx]: remove log

File: app/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      </div>
+      <form
+        onSubmit={handleSubmit(onLoginFinished)}
+        className={classNames(styles.formContainer, styles.fadeIn)}
+      >
+        <span className={styles.first}>
+          Log in to <span className={styles.second}>CHAKRULOS!</span> <br />
+          <div className={styles.container}>
+            <span>
+              New to CHAKRULOS? <span className={styles.signup}>Sign up</span>{" "}
```
Comment (lursmana1) [inline app/Login/Login.tsx]: remove empty string

File: app/Login/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.success) {
+          // Warmatebit Daloginda
+        } else {
+          // Ver gaira login
+        }
+      })
+      .catch((error) => {
+        console.error("Error:", error);
```
Comment (lursmana1) [inline app/Login/Login.tsx]: why console let it be alert

Comment (Dachikkinho) [reply app/Login/Login.tsx]: this is backend code i can delete them, but i think we need this

Comment (lursmana1) [reply app/Login/Login.tsx]: this is your code not backend user can not see error in console let it be alert

**Pull request merged**